app/models/event.js

```diff
@@ -0,0 +1,34 @@
+var Sequelize = require('sequelize');
+
+var schema = {
+	payload: {
+		type: Sequelize.JSONB,
+		allowNull: false
+	},
+	type: {
```

Comment:
Yeah, should be very easy to add that later if needed, since we can just infer based on the event type, if we split them up.